templates/upstart.erb

```diff
@@ -0,0 +1,65 @@
+
+description "<%= @service_name %>"
+
+start on runlevel [2345]
+stop on [!12345]
+
+respawn
+respawn limit 5 60
+
+umask 007
+limit nofile 65536 65536
+limit stack 10240 10240
```

Comment:
Is stack supposed to be bytes and not kbytes? I know ulimit takes kbytes but I think upstart expects bytes. 

Something segfaults when I try to use this config to start Kafka. It works if I remove the `limit stack` line.